lib/lokka/app.rb

```diff
@@ -11,60 +13,86 @@ class App < Sinatra::Base
     configure do
       enable :method_override, :raise_errors, :static, :sessions
       set :app_file, __FILE__
-      set :root, File.expand_path('../../..', __FILE__)
-      set :public_folder => Proc.new { File.join(root, 'public') }
-      set :views => Proc.new { public_folder }
-      set :theme => Proc.new { File.join(public_folder, 'theme') }
-      set :supported_templates => %w(erb haml slim erubis)
-      set :supported_stylesheet_templates => %w(scss sass)
+      set :root, File.expand_path('../..', __dir__)
+      set public_folder: proc { File.join(root, 'public') }
+      set views: proc { public_folder }
+      set theme: proc { File.join(public_folder, 'theme') }
+      set supported_templates: %w[erb haml :slim erubis]
+      set supported_stylesheet_templates: %w[scss sass]
+      set supported_javascript_templates: %w[coffee]
       set :scss, Compass.sass_engine_options
       set :sass, Compass.sass_engine_options
       set :per_page, 10
       set :admin_per_page, 200
       set :default_locale, 'en'
-      set :haml, :ugly => false, :attr_wrapper => '"'
-      set :session_secret, 'development' if development?
+      set :haml, attr_wrapper: '"'
+      set :session_secret, ENV['SESSION_SECRET'] || SecureRandom.hex(30)
+      set :protect_from_csrf, true
       supported_stylesheet_templates.each do |style|
-        set style, :style => :expanded
+        set style, style: :expanded
       end
       ::I18n.load_path += Dir["#{root}/i18n/*.yml"]
-      helpers Lokka::Helpers
-      helpers Lokka::RenderHelper
-      helpers Kaminari::Helpers::SinatraHelpers
-      use Rack::Session::Cookie, {
```

Comment:
As `enable :sessions` are configured, we don't need manual cookie configuration.